spinoffs/inference_gym/inference_gym/targets/dirichlet.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Dirichlet model."""
+
+import numpy as np
+# import tensorflow.compat.v2 as tf
+
+
+# import tensorflow_probability as tfp
+from inference_gym.targets import model
+
+import tensorflow.compat.v2 as tf
+
+import tensorflow_probability as tfp
+from inference_gym.targets import model
+
+
+tfb = tfp.bijectors
+tfd = tfp.distributions
+
+__all__ = [
+    'Dirichlet',
+]
+
+
+class Dirichlet(model.Model):
+  """Creates a Dirichlet.
+
+  
+
+  Note that this function produces reproducible targets, i.e. the constructor
+  `seed` argument always needs to be non-`None`.
+  """
+
+  def __init__(
+      self,
+      concentration_vector=tf.ones(100)+100,
```

Comment:
This arg should be documented in the "Args" section below, also could you add a sentence of so discussing the choice of this default? Like does it produce interesting conditioning etc.